apps/vr-tests-web-components/src/utilities/WCThemeDecorator.tsx

```diff
@@ -2,12 +2,9 @@ import * as React from 'react';
 import { StoryContext } from '@storybook/addons';
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31009&commitId=2d97e683606e80c5e33fe1e9d9fa0a6290364423&env=prod&clientType=FLUENTUI)


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Reply:
This didn't update with the last one - the background *is expected*